httpserver/bin/basic_file_server.dart

```diff
@@ -11,13 +11,12 @@
 import 'dart:io';
 import 'package:http_server/http_server.dart';
 
-void main() {
-
+main() async {
   VirtualDirectory staticFiles = new VirtualDirectory('.');
 
-  HttpServer.bind(InternetAddress.LOOPBACK_IP_V4, 4046).then((server) {
-    server.listen((req) {
-      staticFiles.serveFile(new File('index.html'), req);
-    });
-  });
+  var serverRequests =
```

Comment:
I checked with Kathy. We generally use var. The only time we use types is for clarity or code completion.